packages/editor/src/Sx/Space.tsx

```diff
@@ -67,64 +74,63 @@ export const Space = ({
     }
   }
 
-  const prefixName = name => tag ? `styles.${tag}.${key}${name}` : key + name
-  const label = dir => property === 'margin' ? 'Margin ' + dir : 'Padding ' + dir
+  const prefixName = name => (tag ? `styles.${tag}.${key}${name}` : key + name)
+  const label = dir =>
+    property === 'margin' ? 'Margin ' + dir : 'Padding ' + dir
 
   return (
     <div
       sx={{
         display: 'grid',
-        gridGap: 2,
+        gap: 2,
         gridTemplateColumns: 'repeat(3, 1fr)',
         alignItems: 'center',
       }}>
       <Field
+        // FIXME: Field type is comming from external package @types/theme-ui__components, not sure why type prop does not exist in type definition.
         type="number"
```

Comment:
I get following TypeScript error

```
Type '{ type: string; name: string; label: string; value: any; onChange: (e: any) => void; }' is not assignable to type 'IntrinsicAttributes & FieldOwnProps & Pick<DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement> | ... 19 more ... | PropsWithChildren<...>, "ref" | ... 256 more ... | "sx"> & { ...; }'.
  Property 'type' does not exist on type 'IntrinsicAttributes & FieldOwnProps & Pick<DetailedHTMLProps<HTMLAttributes<HTMLDivElement>, HTMLDivElement> | ... 19 more ... | PropsWithChildren<...>, "ref" | ... 256 more ... | "sx"> & { ...; }'.ts(2322)
```

despite the definition in definitelyTyped being

```ts
export interface InputProps extends Assign<React.ComponentProps<'input'>, BoxOwnProps> {}

[…]

export type FieldProps<T extends React.ElementType> = FieldOwnProps &
    Omit<React.ComponentProps<T>, 'as' | keyof FieldOwnProps> & {
        /**
         * form control to render, default Input
         */
        as?: T;
    };

// `T` is far from unnecessary. We derive component props from it.
// tslint:disable-next-line no-unnecessary-generics
export function Field<T extends React.ElementType = React.ElementType<InputProps>>(props: FieldProps<T>): JSX.Element;
```

I don't understand why this is happening. When I add the prop `as={Input}`, the error goes away. We could use that as a workaround, but that's not particularly nice.

Reply:
Fixed in 196bc704.